root/report/ReleasesConflictingDiscIds.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
```

Comment:
> I didn't update copyright notices on ReportFactory.pm and others, tell me if I should

It isn’t needed except if you want to keep your rights under the copyright, see <https://github.com/metabrainz/musicbrainz-server/pull/1568#discussion_r444129691>. Please confirm you are voluntarily assigning the copyright to the MetaBrainz Foundation or make the relevant changes.

Nitpicking: The title of commit https://github.com/metabrainz/musicbrainz-server/pull/1564/commits/b49e79f29b10ec2a0c74c3f66592abc9168e1c88 is slightly misleading: we might think the linked ticket is about adding subPath property. I found similar examples in your other PRs. Please do not start commit’s message with ticket’s key if it isn’t directly related. If you want to specify this refactoring is required to implement this ticket, please mention ticket’s key later in commit’s message.

Reply:
Force-pushed with fixes to the copyright notice and commit message